src/component-library/CTA/CTA.tsx

```diff
@@ -1,40 +1,42 @@
-import { forwardRef } from 'react';
+import { ButtonHTMLAttributes, forwardRef } from 'react';
 
-import { PrimaryCTA, SecondaryCTA } from './CTA.style';
+import { OutlinedCTA, PrimaryCTA, SecondaryCTA } from './CTA.style';
 
-interface CTAProps extends React.ComponentPropsWithRef<'button'> {
-  disabled?: boolean;
-  variant: 'primary' | 'secondary';
+type Props = {
+  variant?: 'primary' | 'secondary' | 'outlined';
```

Comment:
Regarding types. I strongly suggest we do something more sharable.

Example: https://github.com/nextui-org/nextui/blob/main/packages/react/src/utils/prop-types.ts

Reply:
@tomjeatt @crypto-engineer @peterslany

Reply:
Yes sure, but only if a prop is being shared (which they are in this case). If a prop is specific to a component I'd prefer to keep it with a component. My preference here would be to scope a prop to a component, and move it out to a types file at the point at which it needs to be shared.